users/bescka/index.html

```diff
@@ -60,7 +60,10 @@
   <link rel="icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
   <link rel="shortcut icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
   <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
-  <script src="script.js"></script>
+  <script src="fn/script.js"></script>
```

Comment:
this is absolutely fine as is, but might worth looking into [import/export](https://javascript.info/import-export) and bring in a single script

Reply:
Thanks! Was a bit of a mess - I've refactored the script module logic/ split out and imported in index.html. Much nicer to read! I'll do the same for the other script elements when I get a chance.